apps/parsley/src/context/LogContext/index.tsx

```diff
@@ -150,6 +153,16 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
     ],
   );
 
+  const { settings } = useParsleySettings();
+
+  const sectionsEnabled =
+    isDevelopmentBuild() &&
```

Comment:
I think if you check `!isProduction()` then you can use sectioning on every platform except the actual prod website